atsign-foundation/atmosphere_pro
# UI Redesign - Mobile - Trusted Senders

**- What I did**
**Created a new UI design for the trusted senders screen and a remove trusted sender dialog which closes https://github.com/atsign-foundation/atmosphere_pro/issues/222**

**- How I did it**
**I created a new screen for the trusted senders screen inside the trusted_senders_screen folder**

**- How to verify it**
**add the code to the welcome screen and you should be able to click the  floating action button to view the new screen**
```dart
floatingActionButton: FloatingActionButton(
    backgroundColor: const Color(0xFFF05E37),
    child: const ImageIcon(
        AssetImage("assets/images/add_icon.png"),
        color: Colors.white,
        size: 16.0,
    ),
    onPressed: () {
        showModalBottomSheet<dynamic>(
            isScrollControlled: true,
            shape: RoundedRectangleBorder(
                borderRadius: BorderRadius.circular(20.0),
            ),
            context: context,
            builder: (BuildContext context) {
                return TrustedSendersScreen();
            },
        );
    },
),
```

**- Description for the changelog**
**Added Trusted Senders Screen and Dialog**
